app/build.gradle

```diff
@@ -59,9 +59,11 @@ dependencies {
     // Jetpack Compose
     implementation "androidx.compose.ui:ui:$compose_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
-    implementation 'androidx.compose.material3:material3:1.1.0-alpha04'
+    implementation 'androidx.compose.material3:material3:1.2.0-alpha02'
     debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
     debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_version"
+    implementation "androidx.compose.material:material:$compose_version"
```

Comment:
Why do you need both `material` & `material3`?

Reply:
One of them needed for BottomNavigation (Composable library), and Material3 for Theme implementation